Samples/ShareTarget/cs/ShareTarget.xaml.cs

```diff
@@ -28,9 +29,35 @@
 using Windows.UI.Xaml.Media;
 using Windows.UI.Xaml.Media.Imaging;
 using Windows.UI.Xaml.Navigation;
+using Windows.Web;
 
 namespace SDKTemplate
 {
+    public sealed class StreamUriResolver : IUriToStreamResolver
+    {
+        public IAsyncOperation<IInputStream> UriToStreamAsync(Uri uri)
+        {
+            string path = uri.AbsolutePath;
+            return GetContentAsync(path).AsAsyncOperation();
+        }
+
+        private async Task<IInputStream> GetContentAsync(string URIPath)
+        {
+            try
+            {
+                Uri localUri = new Uri("ms-appdata:///temp/folder/" + URIPath);
```

Comment:
minor things
1) rename URIPath to uriPath (camelCase for parameters)
2) use var for localUri and StorageFile (type name is repeated)